# HG changeset patch
# User Rafael Laboissiere <rafael@debian.org>
# Date 1206549378 -3600
#      Wed Mar 26 17:36:18 2008 +0100
# Node ID 7b382848a18fdaff7968dc2e232d0f13e20d6879
# Parent  ad16ea379d2f3da40ca00b2f6d16c17c2d21d703
Remove spurious whitespace before macros

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,12 @@
+2008-03-26  Rafael Laboissiere  <rafael@debian.org>
+
+	* interpreter/mkoctfile.1: Remove spurious whitespace before macros
+
 2008-03-25  David Bateman  <dbateman@free.fr>
 
 	* interpreter/strings.txi: Document hex2num, num2hex.
 	
 2008-03-21  David Bateman  <dbateman@free.fr>
 
 	* interpreter/sparse.txi: Document amd function.
 
diff --git a/doc/interpreter/mkoctfile.1 b/doc/interpreter/mkoctfile.1
--- a/doc/interpreter/mkoctfile.1
+++ b/doc/interpreter/mkoctfile.1
@@ -102,17 +102,17 @@ Compile or link file. Recognised file ty
     .c    C source
     .cc   C++ source
     .C    C++ source
     .cpp  C++ source
     .f    Fortran source
     .F    Fortran source
     .o    object file
     .a    library file
-    .SH SEE ALSO
-    .BR octave (1).
+.SH SEE ALSO
+.BR octave (1).
 .RE
 .SH AUTHOR
 John W. Eaton <jwe@bevo.che.wisc.edu>
 
 This manual page was contributed by Dirk Eddelbuettel
 <edd@debian.org> for the Debian GNU/Linux distribution but 
 may be used by others.
